example/plasticity/readme.md

```diff
@@ -0,0 +1,23 @@
+# Plasticity Example
+
+A miniapp that demonstrates how to use mutable connectivity. Should be run with
```

Comment:
It looks like the number of cable cells is a variable. You could mention this here, and maybe explain how that number is ultimately set (e.g. by passing a number of ranks to SLURM).

Reply:
The example has been overhauled.